grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/migrate/android/AndroidData.kt

```diff
@@ -75,6 +78,7 @@ internal data class AndroidBinaryData(
     override val databinding: Boolean = false,
     override val compose: Boolean = false,
     override val tags: List<String> = emptyList(),
+    override val lintConfigs: LintConfigs = LintConfigs(),
```

Comment:
We're giving default values to `AndroidBinaryData` but not `AndroidLibraryData`. Would be good to be consistent and giving them all default values?